src/EnergyPlus/ExternalInterface.cc

```diff
@@ -1903,13 +1823,11 @@ std::string trim(std::string const &str)
     return str.substr(first, last - first + 1);
 }
 
-Real64 GetCurSimStartTimeSeconds(EnergyPlusData &state)
+Real64 GetCurSimStartTimeSeconds(const EnergyPlusData &state)
```

Comment:
[src/EnergyPlus/ExternalInterface.cc:1906]:(style),[constParameter],Parameter 'state' can be declared as reference to const